ci/pgbouncer.rb

```diff
@@ -13,7 +12,7 @@ def pgb_rootdir
     task :install do
       Rake::Task['ci:postgres:install'].invoke
       unless Dir.exist? File.expand_path(pgb_rootdir)
-        sh %(wget -O $VOLATILE_DIR/pgbouncer-1.5.4.tar.gz https://s3.amazonaws.com/travis-archive/pgbouncer-1.5.4.tar.gz)
+        sh %(wget -O $VOLATILE_DIR/pgbouncer-1.5.4.tar.gz https://s3.amazonaws.com/dd-agent-tarball-mirror/pgbouncer-1.5.4.tar.gz)
```

Comment:
I think you forgot to c/p the old download link. (and I checked, it's not the first link on Google :stuck_out_tongue: )